javaparser-testing/src/test/java/com/github/javaparser/utils/TestUtils.java

```diff
@@ -12,20 +12,29 @@
 import java.util.zip.ZipInputStream;
 
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 public class TestUtils {
     /**
      * Takes care of setting all the end of line character to platform specific ones.
      */
-    public static String readFileWith(String resourceName) throws IOException {
-        try (final InputStream inputStream = TestUtils.class.getClassLoader().getResourceAsStream(resourceName);
-             final BufferedReader br = new BufferedReader(new InputStreamReader(inputStream, "utf-8"))) {
-            final StringBuilder builder = new StringBuilder();
-            String line;
-            while ((line = br.readLine()) != null) {
-                builder.append(line).append(Utils.EOL);
+    public static String readResource(String resourceName) throws IOException {
```

Comment:
I threw this into the project a little while ago, I think I had a similar problem to solve :)